python/cudf/cudf/_libxx/slice.pyx

```diff
@@ -0,0 +1,140 @@
+# Copyright (c) 2018-2020, NVIDIA CORPORATION.
```

Comment:
This file should be in a `strings` directory to indicate it's specifically for string functions